.github/ISSUE_TEMPLATE/1_bug_report.yaml

```diff
@@ -0,0 +1,98 @@
+name: Report a bug
+description: Any errors that are not UI related.
+labels: ["needs triage", "bug"]
```

Comment:
@Felonious-Spellfire This should be kept as just "needs triage" as it was in the previous file. Often users misclassify as bugs, and all issues need to be triaged anyways by a maintainer.